.devcontainer/default/VSCode/settings.json

```diff
@@ -0,0 +1,59 @@
+{
+  // -- CMake Tools
+  "cmake.sourceDirectory": "${workspaceFolder}/llvm/llvm",
+  "cmake.buildDirectory": "${workspaceFolder}/build/circt",
+  "cmake.installPrefix": "/usr/local",
+  "cmake.configureOnOpen": false,
+  "cmake.configureOnEdit": false,
+  "cmake.generator": "Ninja",
+  "cmake.configureArgs": [
+    // CMAKE
+    "-DCMAKE_WARN_DEPRECATED=OFF",
+    "-DCMAKE_C_COMPILER_LAUNCHER=ccache",
+    "-DCMAKE_CXX_COMPILER_LAUNCHER=ccache",
+
+    // LLVM
+    "-DLLVM_ENABLE_ASSERTIONS=ON",
+    "-DLLVM_ENABLE_PROJECTS=mlir",
+    "-DLLVM_TARGETS_TO_BUILD=host",
+    "-DLLVM_BUILD_EXAMPLES=OFF",
+    "-DLLVM_ENABLE_OCAMLDOC=OFF",
+    "-DLLVM_ENABLE_BINDINGS=OFF",
+    "-DLLVM_ENABLE_MODULES=OFF",
+    "-DLLVM_USE_SANITIZER=",
+
+    // LLVM (performance)
+    "-DLLVM_USE_LINKER=lld-17",
```

Comment:
is `lld-17` good here, or should this be set/substituted in?  Similarly, `clangd-17` below.

Reply:
Oh generally looks like this hardcodes things that are variables elsewhere (e.g., build paths).  Should this be generated instead?